app/javascript/components/Games.jsx

```diff
@@ -23,7 +23,7 @@ export const Games = ({endpoint, loading, user, homeError = null}) => {
     const [numCategories, setNumCategories] = useState(0)
 
     useEffect(() => {
-        getData(`/${user ? 'user_games' : 'games'}${search ? `?name=${search}`: ''}`, setData, homeError || setError)
+        getData(`${user ? 'user_games' : 'games'}${search ? `?name=${search}`: ''}`, setData, homeError || setError)
```

Comment:
I see that you are simply removing a `/` from this endpoint string, so perhaps this is outside of the scope of this refactor. However, I wanted to mention that this string being built takes some time to wrap my mind around. It might be best to pull some of these things out into namable variables to help make parsing this string easier on the reader.
```suggestion
        const url = user ? 'user_games' : 'games'
        if (search) {
          url += `?name=${search}`
        }
        
        getData(url, setData, homeError || setError)
```

Reply:
that's helpful input, thanks! I definitely see how this can be tedious to understand as written; I'll look into clarifying this in a future PR